financial-services-accelerator/react-apps/self-care-portal/src/main/java/org/wso2/financial/services/accelerator/scp/webapp/util/Utils.java

```diff
@@ -163,27 +169,33 @@ public static LocalDateTime parseEncodedStringToDate(String dateStr) {
     @Generated(message = "Ignoring since method do not contain a logic")
     public static void returnResponse(HttpServletResponse resp, int statusCode, JSONObject payload) {
         try {
-            LOG.debug("Returning response to frontend. payload: " + payload.toString());
+            LOG.debug(String.format("Returning response to frontend. payload: %s",
+                    payload.toString().replaceAll("\n\r", "")));
             // returning  response
             resp.setContentType(ContentType.APPLICATION_JSON.getMimeType());
             resp.setStatus(statusCode);
             PrintWriter out = resp.getWriter();
-            out.print(payload);
+//            out.print(payload);
+            out.write(Encode.forHtml(payload.toString()));
             out.flush();
         } catch (IOException e) {
             LOG.error("Exception occurred while returning response. Caused by, ", e);
         }
     }
 
+    @SuppressFBWarnings("FORMAT_STRING_MANIPULATION")
+    // Suppressed content - final String errorUrl = String.format(errRedirectUrlFormat, errorMsg, errorDesc);
+    // Suppression reason - False Positive : These fields are internally generated.
+    // Suppressed warning count - 1
     @Generated(message = "Ignoring since method do not contain a logic")
     public static void sendErrorToFrontend(SelfCarePortalError error, String errRedirectUrlFormat,
                                            HttpServletResponse resp) {
         LOG.debug("Sending error to frontend.");
         try {
             final String errorMsg = Base64.getUrlEncoder()
-                    .encodeToString(error.getMessage().getBytes(StandardCharsets.UTF_8));
+                    .encodeToString(error.getMessage().getBytes(StandardCharsets.UTF_8)).replaceAll("\n\r", "");
```

Comment:
Shall we use a constant for "\n\r"?

Reply:
FIxed in https://github.com/wso2/financial-services-accelerator/pull/752/commits/1e2edab95303b5a528064f8a4ceb3acb3d75fd36